client/src/components/RequestForm/RequestForm.tsx

```diff
@@ -0,0 +1,156 @@
+                        value={props.values.dropInDate}
+                        renderInput={(params) => <TextField sx={{ width: '60%' }} {...params} />}
+                      />
+                      <Autocomplete
+                        onChange={(event, value) => {
+                          props.setFieldValue('dropInTime', value);
+                        }}
+                        defaultValue={props.values.dropInTime}
+                        disableClearable
+                        options={getTimes(
+                          props.values.dropInDate.toDateString() === today.toDateString()
+                            ? props.values.dropInDate.getHours()
+                            : 0,
+                        )}
+                        renderInput={(params) => <TextField {...params} />}
+                        sx={{ width: '40%' }}
+                      />
+                    </Box>
+                  </Box>
+                  <Box display="flex" flexDirection="column" marginTop={3}>
+                    <InputLabel className={classes.label} sx={{ fontWeight: 700, color: '#000' }}>
+                      Drop Off
+                    </InputLabel>
+                    <Box display="flex">
+                      <DatePicker
+                        onChange={(value) => props.setFieldValue('dropOffDate', value)}
+                        value={props.values.dropOffDate}
+                        renderInput={(params) => <TextField sx={{ width: '60%' }} {...params} />}
+                      />
+                      <Autocomplete
+                        onChange={(event, value) => props.setFieldValue('dropOffTime', value)}
+                        defaultValue={props.values.dropOffTime}
+                        disableClearable
+                        options={getTimes(
+                          props.values.dropOffDate.toDateString() === today.toDateString()
+                            ? props.values.dropOffDate.getHours() + 1
+                            : 0,
+                        )}
+                        renderInput={(params) => <TextField {...params} />}
+                        sx={{ width: '40%' }}
+                      />
+                    </Box>
+                  </Box>
+
+                  <Box textAlign="center" marginTop={5}>
+                    <Button
+                      type="submit"
+                      size="large"
+                      variant="contained"
+                      color="primary"
+                      className={classes.submit}
+                      disableElevation
+                    >
+                      {props.isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Send request'}
+                    </Button>
+                  </Box>
+                </Form>
+              )}
+            </Formik>
+          </LocalizationProvider>
+        </Grid>
+      </Grid>
+    </Card>
+  );
+};
+
+const getRankInStar = (rank: number): JSX.Element => {
```

Comment:
Good thinking! Why don't we use the `Rating` component from MUI instead 

Ref: https://mui.com/components/rating/